quarkchain/cluster/tests/test_jsonrpc.py

```diff
@@ -57,6 +65,57 @@ def send_request(*args):
     return call_async(__send_request(*args))
 
 
+@contextmanager
+def jrpc_websocket_server_context(slave_server):
+    env = DEFAULT_ENV.copy()
+    env.cluster_config = ClusterConfig()
+    env.cluster_config.JSON_RPC_PORT = 38391
+    # to pass the circleCi
+    env.cluster_config.JSON_RPC_HOST = "127.0.0.1"
+
+    slave_config = SlaveConfig()
+    slave_config.HOST = "0.0.0.0"
+    slave_config.PORT = 38001
+    slave_config.WEBSOCKET_JSON_RPC_PORT = 38591
+    slave_config.ID = "S1"
+    slave_config.CHAIN_MASK_LIST = [5]
+
+    env.cluster_config.SLAVE_LIST.append(slave_config)
+    env.slave_config = env.cluster_config.get_slave_config("S1")
+    server = JSONRPCWebsocketServer.start_websocket_server(env, slave_server)
+    try:
+        yield server
+    finally:
+        server.shutdown()
+
+
+def send_websocket_request(request):
+    responses = []
+
+    async def __send_request(request):
+        uri = "ws://0.0.0.0:38591"
+        async with websockets.connect(uri) as websocket:
+            await websocket.send(request)
+            while True:
+                response = await websocket.recv()
+                responses.append(response)
+                if len(responses) == 2:
```

Comment:
2 can be passed in as an argument in `send_websocket_request` with a default value (like 1)